cmd/status/status.go

```diff
@@ -72,6 +73,10 @@ func runStatus() error {
 		ss.NumFlows,
 		(float64(ss.NumFlows)/float64(ss.MaxFlows))*100,
 	)
+	if ss.UptimeNs > 0 {
+		fmt.Printf("Flows/s: %.2f\n", float64(ss.SeenFlows)/time.Duration(ss.UptimeNs).Seconds())
```

Comment:
i'm assuming unlikely, but still.. divide by zero guard?

Reply:
I guess I went for the same approach as we did with the unavailable nodes: If it's not part of the status message (i.e. Hubble v0.5 server) we omit it. But I can add the `N/A`.

As for the guard: Good catch. I guess it won't cause a crash (non-signalling float math), but `Flows/s: Inf` will not be a good result either :D

Reply:
Addressed.

Reply:
Oh, actually, 1 ns is still too large ever cause an float underflow. So the previous check could never fail as well, but let's guard on the `Seconds()` anyways.